libraries/Networking/src/ESP_Network_Interface.h

```diff
@@ -0,0 +1,73 @@
+#pragma once
+
+#include "esp_netif_types.h"
+#include "esp_event.h"
+#include "Arduino.h"
+#include "ESP_Network_Manager.h"
+
+typedef enum {
+    ESP_NETIF_ID_STA,
+    ESP_NETIF_ID_AP,
+    ESP_NETIF_ID_PPP,
+    ESP_NETIF_ID_ETH0,
+    ESP_NETIF_ID_ETH1,
+    ESP_NETIF_ID_ETH2,
+    ESP_NETIF_ID_MAX
+} ESP_Network_Interface_ID;
+
+#define ESP_NETIF_ID_ETH ESP_NETIF_ID_ETH0
+
+class ESP_Network_Interface {
```

Comment:
Are these underscores standard in class names ... most other classes do not use underscores?

The Arduino Style Guide (https://docs.arduino.cc/learn/contributions/arduino-library-style-guide/) does not specifically mention this or have examples, but it does refer to https://processing.org/reference in the context of camel case function names.  Processing.org has title case class names, e.g. ArrayList.